docs/root/configuration/http/http_filters/aws_lambda_filter.rst

```diff
@@ -22,7 +22,8 @@ If :ref:`payload_passthrough <envoy_v3_api_field_extensions.filters.http.aws_lam
 However, if :ref:`payload_passthrough <envoy_v3_api_field_extensions.filters.http.aws_lambda.v3.Config.payload_passthrough>`
 is set to ``false``, then the HTTP request is transformed to a JSON payload with the following schema:
 
-.. code-block::
+.. code-block:: json
+   :force:
```

Comment:
requires css hack

Reply:
Can you unbundle the changes with interesting things from the `**` --> quotes mass change. Hard to spot what to pay attention to.

Reply:
yep, that was kinda my intention

Reply:
code changes are here and here:

- #22198 
- #22197